import exception_4_class
import fib

print(exception_4_class.great('Red'))
print(fib.fib(5))